test/membrane_live_web/controllers/user_controller_test.exs

```diff
@@ -56,28 +68,45 @@ defmodule MembraneLiveWeb.UserControllerTest do
       conn = put(conn, Routes.user_path(conn, :update, user), user: @invalid_attrs)
       assert json_response(conn, 422)["errors"] != %{}
     end
+
+    test "reject if updating not the given user", %{conn: conn, user: user} = context do
+      callback = &put(&1, Routes.user_path(conn, :update, user), user: @update_attrs)
+      test_unauthorized_user_request(callback, context)
+    end
   end
 
   describe "delete user" do
-    setup [:create_user]
-
     test "deletes chosen user", %{conn: conn, user: user} do
       conn = delete(conn, Routes.user_path(conn, :delete, user))
       assert response(conn, 204)
 
       conn = get(conn, Routes.user_path(conn, :show, user))
       assert response(conn, 404)
     end
+
+    test "reject if deleting not the given user", %{conn: conn, user: user} = context do
+      conn_callback = &delete(&1, Routes.user_path(conn, :delete, user))
+      test_unauthorized_user_request(conn_callback, context)
+    end
   end
 
-  defp create_user(_user) do
-    user = user_fixture()
-    %{user: user}
+  defp test_unauthorized_user_request(conn_callback, %{
+         conn: conn,
+         user: %{uuid: user_uuid}
+       }) do
+    fake_user = fake_user_fixture()
+    expected_msg = unauthorized_error_message(fake_user.uuid, user_uuid)
+
+    conn
+    |> put_user_in_auth_header(fake_user)
+    |> then(conn_callback)
+    |> unauthorize_assert(expected_msg)
   end
 
-  defp get_valid_bearer() do
-    with {:ok, valid_token, _claims} <- Tokens.auth_encode(@dummy_uuid) do
-      "Bearer #{valid_token}"
-    end
+  def unauthorize_assert(conn, expected_msg) do
+    assert %{"message" => ^expected_msg} = json_response(conn, 403)
   end
+
+  defp unauthorized_error_message(fake_id, user_id),
+    do: "User with uuid #{fake_id} does not have access to user with uuid #{user_id}"
```

Comment:
It looks very similiar to the same function with same name in `webinar_controller_test.exs`.